frontend/src/hooks/useFetchKeywords.ts

```diff
@@ -1,12 +1,12 @@
-import axios from "axios";
-import toast from "react-hot-toast";
 import { useContext } from "react";
 import { LayoutContext } from "../components/layouts";
 import { useResults } from "./useResults";
 import { useBlockUi } from "./useBlockUi";
 import { formatSorting, removeEmptyObj } from "../helper/format";
 import { SortingState } from "@tanstack/react-table";
 import { useModalResult } from "./useModalResult";
+import request from "../lil/request";
+import myToast from "../lil/toast";
```

Comment:
`my` suffix could be misleading. A better prefix could be `app`:

```
import appToast from "../lil/toast";
```

Or simply not using any prefix as done for `request` 😅